README.md

```diff
@@ -192,3 +195,33 @@ Has SweetAlert2 helped you create an amazing application? You can show your supp
 - https://www.patreon.com/limonte
 
 ### [Hall of Donators :trophy:](DONATIONS.md)
+
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```

Comment:
We have no `CONTRIBUTING.md` but we have a Contributing section in this README.md. This could link to `#contributing`